gulpfile.ts

```diff
@@ -9,7 +9,7 @@ loadTasks(PROJECT_TASKS_DIR);
 
 // --------------
 // Build dev.
-gulp.task('build.dev', done =>
+gulp.task('build.dev', (done: any) =>
```

Comment:
Could be a nice solution.
Only used `any` cause I couldn't find a better solution.